src/sysv-generator/sysv-generator.c

```diff
@@ -312,7 +314,7 @@ static int sysv_translate_facility(SysvStub *s, unsigned line, const char *name,
                 if (r < 0)
                         return log_error_errno(r, "[%s:%u] Could not build name for facility %s: %m", s->path, line, name);
 
-                return r;
+                return 1;
```

Comment:
Seems like `sysv-generator-test.py` doesn't test that case.
Why not to add
```diff
diff --git a/test/sysv-generator-test.py b/test/sysv-generator-test.py
index 16ea656..8411381 100755
--- a/test/sysv-generator-test.py
+++ b/test/sysv-generator-test.py
@@ -424,5 +424,14 @@ class SysvGeneratorTest(unittest.TestCase):
         self.assertEqual(os.listdir(self.out_dir), [])


+    def test_target_facility(self):
+        '''single LSB macro dependency: $multi-user'''
+
+        self.add_sysv('foo', {'Required-Start': '$multi-user'})
+        s = self.run_generator()[1]['foo.service']
+        self.assertEqual(set(s.options('Unit')),
+                         set(['Documentation', 'SourcePath', 'Description', 'After']))
+        self.assertEqual(s.get('Unit', 'After'), 'multi-user.target')
+
 if __name__ == '__main__':
```
?
@martinpitt , what do you think?